apps/mobile/src/components/ContextMenu/ContextMenu.js

```diff
@@ -96,7 +96,15 @@ function MenuItem ({ widget, groupSlug, rootPath, group }) {
   }, [url, routeParams.originalLinkingPath])
 
   const handleWidgetPress = widget => {
+    if (widget?.customView?.externalLink) {
+      try {
+        openURL(ensureHttpsForPath(widget.customView.externalLink))
+      } catch (error) {
```

Comment:
If we want to handle the error and just have it console.log then I'd add that to openURL. I think I'd prefer to just not handle this error though so it passes through to Sentry error capture.